### PR TITLE
Feat: Support metadata topics in nova visualizer

### PR DESCRIPTION
**Pull request by msarcev**
# Description of change

- Sets new Visualizer to be used only on Nova networks
- Listens to `block-metadata` topics and streams updates in novaFeed
- Colours Blocks in the Visualizer based on `TransactionState` 

Closes https://github.com/iotaledger/explorer/issues/1034

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: client/src/features/visualizer-threejs/VisualizerInstance.tsx [hunk 1]
```diff
@@ -198,7 +206,9 @@ const VisualizerInstance: React.FC<RouteComponentProps<VisualizerRouteProps>> =
         if (!feedServiceRef.current) {
             return;
         }
-        feedServiceRef.current.subscribeBlocks(onNewBlock, onBlockMetadataUpdate);
+        const novaFeedService = feedServiceRef.current as NovaFeedClient;
```
Comment (VmMad) [inline client/src/features/visualizer-threejs/VisualizerInstance.tsx:209]: I think this line is unnecessary

**Review by VmMad — APPROVED** [review]
Looks good to me

File: client/src/features/visualizer-threejs/constants.ts [hunk 2]
```diff
@@ -22,19 +23,16 @@ export const ANIMATION_TIME_SECONDS = 3;
...
-    new Color("#5C84FA"),
-    new Color("#2559F5"),
-    ...ACCEPTED_BLOCK_COLORS,
-];
+export const ACCEPTED_BLOCK_COLOR = new Color("#0101AB");
+export const CONFIRMED_BLOCK_COLOR = new Color("#0000DB");
+export const FINALIZED_BLOCK_COLOR = new Color("#0101FF");
+// TODO Remove accepted state once is added to the SDK (missing)
+export const BLOCK_STATE_TO_COLOR = new Map<BlockState | "accepted", Color>([
+    ["pending", PENDING_BLOCK_COLOR],
```
Comment (begonaalvarezd) [inline client/src/features/visualizer-threejs/constants.ts:31]: couldnt these colors be written as `BlockState.Pending` so we avoid magic strings? 
```suggestion
    [BlockState.Pending, PENDING_BLOCK_COLOR],
```

Comment (msarcev) [reply client/src/features/visualizer-threejs/constants.ts:31]: Unfortunately not cause it's not an ENUM, but a string union type.
It is type-safe though, if you try to use a value that is not part of `BlockState` TS will complain.

Comment (VmMad) [reply client/src/features/visualizer-threejs/constants.ts:31]: Yeah, seems like doing an enum would be redundant

**Pull request merged**